manifest.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Very cool idea, love it! FYI it only worked when running it with the Ruby server. It'll probably work better on https://ledmatrix.frame.work/.

Reply:
Yeah, I'd assume that's a limitation of how PWAs work. If someone downloads the files and uses the download, I think it's safe to assume they don't need the PWA installed then.